### DISCUSSION TITLE
[Hungarian] ocular translation tooltip has html tag

### DISCUSSION THREAD

**Discussion by ghost** [ScratchAddons/ScratchAddons#7516] [Translation] [answered]
![kép](https://github.com/ScratchAddons/ScratchAddons/assets/150537842/5f941735-b174-46a0-9c78-611aa2c97794)
I think I'm fine to put this here, but not sure if I am.
Sorry, I just don't really wanna join Transifex.
The string in question should be `my-ocular/status-hover`.

**Comment by WorldLanguages** [comment]
I can change it

**Reply by ghost** [reply]
Please do change it

**Reply by WorldLanguages** [reply, marked as answer]
Done

**Reply by ghost** [reply]
thank you!!

**Reply by WorldLanguages** [reply]
I mean, it should come up in tomorrow's localization update PR

**Reply by ghost** [reply]
> I mean, it should come up in tomorrow's localization update PR

its fine if its delayed
its not a really big thing
